apps/web/src/server/applications/create/case/__tests__/applications-create-case.test.js

```diff
@@ -167,6 +167,105 @@ describe('applications create', () => {
 				});
 			});
 		});
+
+		describe('GET /create-new-case/1', () => {
+			const baseUrl = '/applications-service/create-new-case/1';
+
+			beforeEach(async () => {
+				await request.get('/applications-service/case-officer');
+				nocks();
+			});
+
+			it('should render page with resumed data', async () => {
+				const response = await request.get(baseUrl);
+				const element = parseHtml(response.text);
+
+				expect(element.innerHTML).toMatchSnapshot();
+				expect(element.innerHTML).toContain(fixtureApplications[0].description.slice(0, 20));
+			})
+		})
+
+		describe('POST /create-new-case/:id', () => {
+			const baseUrl = '/applications-service/create-new-case';
+
+			beforeEach(async () => {
+				await request.get('/applications-service/case-officer');
```

Comment:
We probably want to remove these Web changes

Reply:
yes, there are some files from other commits - should I remove all of these?

Reply:
If you did not mean to make them and they are a result of the rebasing, then we want to make sure that we are pushing the appropriate changes. I would check what this file looks like in the BOAS-166 branch and just copy it over.